lang/interpolate_test.go

```diff
@@ -127,6 +128,66 @@ func TestInterpolate0(t *testing.T) {
 			ast:  ast,
 		})
 	}
+	{
+		ast := &StmtProg{
+			Prog: []interfaces.Stmt{
+				&StmtRes{
+					Kind: "test",
+					Name: &ExprStr{
+						V: "t1",
+					},
+					Contents: []StmtResContents{
+						&StmtResField{
+							Field: "stringptr",
+							Value: &ExprStr{
+								V: "${hello}",
+							},
+						},
+					},
+				},
+			},
+		}
+		testCases = append(testCases, test{
+			name: "variable escaping 1",
+			code: `
+			test "t1" {
+				stringptr => "\${hello}",
```

Comment:
This relies on external behaviours of the implicit global variables (which I think should probably be removed). Can you add an explicit `$hello = "hello"` above?

Reply:
Not sure what you mean...

Reply:
So these are only parse tests, not parse+eval tests. You can ignore me :)
I thought these were referring to some static variables in the code that I now can't find